students/douglas_klos/session3/mailroom.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python3
+
+# Douglas Klos
+# January 28th, 2019
+# Python 210
+# mailroom.py
+
+# There are more functions than requested in the assignment.
+# I felt it needed a better way to add new donors than just through
+# send_thank_you function, and a way to add new donations to current
+# donors.  Note that you can add new donors through send_thank_you
+# as requested in the assignment.
+
+
+mailroom_db = [('Douglas', [5000, 2000]),
+               ('Maggie', [2222, 3333, 4444]),
+               ('Light Yagami', [124, 8975]),
+               ('Makise Kurisu', [235987]),
+               ('Youjo Senki', [13498, 9876, 1234]),
+               ('Motoko Kusanagi', [57892, 239857, 87265]),
+               ('Jo', [8814, 2320])
+               ]
+
+MAIN_PROMPT = '\n'.join((
+              '\nWelcome to the Mail Room!',
+              'Please choose from the following options: ',
+              '1: Send a Thank You',
+              '2: Create a report',
+              '3: Add a new donor',
+              '4: Add a new donation',
+              'q: Quit',
+              '>>> '))
+
+THANK_YOU_PROMPT = '\n'.join((
+                   '\nPlease enter one of the following:',
+                   '<Name>: Name of person to send a thank you note to',
+                   'list: Display donors in the database',
+                   'q: Return to main menu ',
+                   '>>> '))
+
+
+THANK_YOU_NOTE = '\n'.join((
+                 '\nDear {}:',
+                 '\tThank you for your very kind donation of ${:.2f}.',
+                 '\tIt will be put to very good use.',
+                 '\t\tSincerely',
+                 '\t\t\tThe Team'))
+
+
+def add_donation():
+    """ Add new donations to a current donor """
+
+    display_database()
+    while True:
+        name = input('\nPlease select a donor to add a donation to: ')
+        if name.lower() in ('q', 'quit'):
+            return
+        elif not any(name in sub_list for sub_list in mailroom_db):
```

Comment:
you'll be glad when we restructure to use dictionaries and you don't have to do this :)